_includes/technology/repos.md

```diff
@@ -1,21 +1,19 @@
 ## Repositories
 
-There are many individual repositories that make up eRegulations; this is
-complicated by having two primary forks maintained by CFPB and 18F
-respectively. While these forks will ultimately be merged and maintenance
-shared between multiple agencies, those interested in using eRegs now should
-generally prefer the 18F repositories.
+There are many individual repositories that make up eRegulations. The project is divided into several components, and currently CFPB and 18F maintain two separate forks of **regulations-parser**, **regulations-core**, and **regulations-site**. 18F and CFPB plan to eventually merge these forks into shared components with maintenance shared between multiple agencies. For now, in most cases people interested in using eRegulations should use the 18F repositories (listed under "General Purpose").
+
+*Tip: If you'd simply like to get a copy of eRegulations set up to try it out, try starting with [CFPB **regulations-bootstrap**](https://github.com/cfpb/regulations-bootstrap) or [ATF **atf-eregs**](https://github.com/18F/atf-eregs), which are both also listed below.*
```

Comment:
I don't think this is a good idea just yet. The bootstrap repo pulls in CFPB-centric parsing code (which won't work for other agencies) and ATF-eRegs doesn't include _any_ parsing code and would certainly be confusing: "why do I want ATF's UI?").

We definitely need to think through a gentler introduction to eRegs from a technical side, but let's hold off on directing new folks to misleading repos until then.